### PR TITLE
mount path in DS projects

### PR DESCRIPTION
**Pull request by pnaik1**
https://issues.redhat.com/browse/RHOAIENG-426

## Description

- remove add storage button
- Adding icon before storage name
- Rename workbench storage to Cluster storages
- Full mount path
- show all button

Screenshot 2024-10-11 at 12 32 35 PM

## How Has This Been Tested?

- Go to Workbenches tab from project page
- Click on one of the notebook/workbench row
- Check whether mount path has full path
- Check show less/all button

## Test Impact
Added cypress test

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/pages/projects/screens/detail/notebooks/ShowAllButton.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import React from 'react';
+
+type ShowAllButtonProps = {
+  visibleLength: number;
+  isExpanded: boolean;
+  totalSize: number;
+  onToggle: () => void;
+};
+
+const ShowAllButton: React.FC<ShowAllButtonProps> = ({
```
Comment (pnaik1) [inline frontend/src/pages/projects/screens/detail/notebooks/ShowAllButton.tsx:11]: FYI, this separate button component is created  to have identical function for both cluster storage and data connection as mentioned in the mock

File: frontend/src/pages/projects/screens/detail/notebooks/NotebookStorageBars.tsx [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
-const NotebookStorageBars: React.FC<NotebookStorageBarsProps> = ({ notebook, onAddStorage }) => {
+const NotebookStorageBars: React.FC<NotebookStorageBarsProps> = ({ notebook }) => {
   const [pvcs, loaded, loadError] = useNotebookPVCItems(notebook);
   const mountFolderMap = getNotebookPVCMountPathMap(notebook);
+  const [showAll, setShowAll] = React.useState(false);
+  const defaultVisibleLength = 2;
+  const visiblePvcs = showAll ? pvcs : pvcs.slice(0, defaultVisibleLength);
 
   return (
     <Stack>
```
Comment (DaoDaoNoCode) [inline frontend/src/pages/projects/screens/detail/notebooks/NotebookStorageBars.tsx:36]: I think you will need a `hasGutter` here after adding the show all button to make the layout look better.

**Comment by xianli123** [comment]
Thanks @pnaik1 @DaoDaoNoCode ! LGTM!

File: frontend/src/pages/projects/screens/detail/notebooks/NotebookStorageBars.tsx [hunk 3]
```diff
@@ -39,40 +46,51 @@ const NotebookStorageBars: React.FC<NotebookStorageBarsProps> = ({ notebook, onA
...
-                        : 'Unknown'}
-                    </Text>
-                  </StackItem>
-                </Stack>
+                    <Stack>
+                      <StackItem>
+                        <StorageSizeBar pvc={pvc} />
+                      </StackItem>
+                      <StackItem>
+                        <Text component="small">
```
Comment (Gkrumbach07) [inline frontend/src/pages/projects/screens/detail/notebooks/NotebookStorageBars.tsx:66]: these texts should not be small anymore, the mocks show them as the same size as the others

**Review by Gkrumbach07 — CHANGES_REQUESTED** [review]

**Review by Gkrumbach07 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**